packages/Uploader/src/components/File/File.js

```diff
@@ -100,16 +104,24 @@ function File(props) {
       case types.status.IDLE:
         return I18n.t("uploader.progress.idle");
       default:
-        return I18n.t("uploader.progress.uploading", { progressWithUnits, sizeWithUnits });
+        if (!showA11yProgress) return I18n.t("uploader.progress.uploading", { progressWithUnits, sizeWithUnits });
+        return I18n.t("uploader.progress.uploading_percent", { a11yProgress });
     }
   }
 
   return (
-    <sc.File>
+    <sc.File
+      aria-label={I18n.t("uploader.progress.file_progress", { name, progress: getProgressText(true) })}
+      aria-valuemin={0}
+      aria-valuemax={100}
+      aria-valuenow={progress}
+      role="progressbar"
+      tabIndex={0}
```

Comment:
Do we still need this? The list item itself is not interactive, so it may not need keyboard focus, especially if the `aria-live` field is announcing properly.

Reply:
Hmm, without it and with just `aria-live` it will constantly announce changes in upload status like "uploading X KiB or Y MiB" until X = Y. Also, the announcements are a bit buggy for me, sometimes thing aren't announced when finished (but if you tab across the progress bars, it will announce the status).

Reply:
Yeah, I'm not getting the right `aria-live` behaviour I'd like either.  I'll play with it a bit more tomorrow.